charts/portworx/values.yaml

```diff
@@ -1,8 +1,12 @@
 # Please uncomment and specify values for these options as per your requirements.
 
+deployOperator: true                    # Deploy the Portworx operator
+deployCluster: true                     # Deploy the Portworx cluster
+
 deploymentType: oci                     # accepts "oci" or "docker"
 imageType: none                         #
 imageVersion: 2.9.1.4                   # Version of the PX Image.
```

Comment:
Should we change to 2.10.x?

Reply:
If this is the case, we need to create different helm chart release version like what we do for the PSO. Customers won't know which version to specify. @piyush-nimbalkar : what's your suggestion?

Reply:
I will bump up the version of both in another PR.

Reply:
@dsupure Looks good what @haibinxie is suggesting. We generally bump up the chart version with the default px version. 
About the version, we want to stick to 2.9.1.4 because that is the default version in our spec generator too - https://install.portworx.com/version. When we update the spec gen, we can update the helm chart too.